packages/flutter/lib/src/material/text_button.dart

```diff
@@ -105,6 +105,20 @@ class TextButton extends ButtonStyleButton {
   ///
   /// {@macro flutter.material.ButtonStyleButton.iconAlignment}
   ///
+  /// {@tool dartpad}
```

Comment:
IIRC `{@tool dartpad}` requires a standalone runnable Dart file. This kind of snippet is used with `{@tool snippet}`. If you'd like a runnable dartpad, refer to other `{@tool dartpad}` blocks for how it's done (a separate file along with a test file).